server/server.go

```diff
@@ -52,6 +52,14 @@ type Server struct {
 
 	// Log is the log15.Logger for the service.  You may replace or call SetHandler() on this at any time to change the logging of the service.
 	Log log15.Logger
+
+	// Metrics is optional and when provided ari-proxy invokes OnXXX when an event or request is received
+	Metrics Metrics
```

Comment:
For some reason, this metric has been removed. I was trying to move on without this, but we've already used this Metric a lot. So, copied it from the below.

https://github.com/AVOXI/ari-proxy/blob/9e40ac8a2b2f231a8b40f4d2db45c5c3796c5316/server/server.go#L57